x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -442,10 +440,34 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
     return this._subPlugins;
   }
 
+  /**
+   * All started subPlugins.
+   */
+  private async startSubPlugins(
+    storage: Storage,
+    core: CoreStart,
+    plugins: StartPlugins
+  ): Promise<StartedSubPlugins> {
+    const subPlugins = await this.subPlugins();
+    return {
+      overview: subPlugins.overview.start(),
+      detections: subPlugins.detections.start(storage),
+      cases: subPlugins.cases.start(),
+      hosts: subPlugins.hosts.start(storage),
+      network: subPlugins.network.start(storage),
+      timelines: subPlugins.timelines.start(),
+      management: subPlugins.management.start(core, plugins),
+    };
+  }
+
   /**
    * Lazily instantiate a `SecurityAppStore`. We lazily instantiate this because it requests large dynamic imports. We instantiate it once because each subPlugin needs to share the same reference.
    */
-  private async store(coreStart: CoreStart, startPlugins: StartPlugins): Promise<SecurityAppStore> {
+  private async store(
+    coreStart: CoreStart,
+    startPlugins: StartPlugins,
+    subPlugins?: StartedSubPlugins
```

Comment:
I added the `subPlugins` parameter as optional right now so old sub-plugins continue working. 
When all sections are migrated we'll just have to make it required and remove all those `subPlugins ?` conditionals.